patterns-base/src/main/scala/codacy/patterns/Custom_Scala_GetCalls.scala

```diff
@@ -0,0 +1,24 @@
+package codacy.patterns
+
+import codacy.base.Pattern
+
+import scala.meta._
+
+case object Custom_Scala_GetCalls extends Pattern {
```

Comment:
Codacy found an issue: [Object name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-scalameta/pullRequest?prid=6180926)